### DISCUSSION TITLE
where does one find the schema for the table-of-contents file?

### DISCUSSION THREAD

**Discussion by nmaylevc** [CMSgov/price-transparency-guide#511] [Q&A] [answered]
I see a description of the table-of-contents file and an example file, but I can't find the schema for that file.
Where is it?
A published schema would be useful and if it would be great if it were in the "Schemas" section of the document.

**Comment by RLTx1391** [comment, marked as answer]
https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/table-of-contents

**Reply by nmaylevc** [reply]
thank you.  It would be clear if that page was linked to from the Schemas section of the top level doc.  I don't see it.

**Reply by shaselton-usds** [reply]
All schemas are within the schema directory -- pretty clear.

**Reply by nmayle** [reply]
It would be even clearer if they were linked to from the Schemas section of the main page (https://github.com/CMSgov/price-transparency-guide). That section only lists two schemas. Why those two and not the others?

**Comment by nmaylevc** [comment]
Adding links to them to this list would be useful:

https://github.com/CMSgov/price-transparency-guide
